### PR TITLE
Added Ubuntu 14.04 compatibility #27

### PR DESCRIPTION
**Pull request by catalinvr**
This fixes the #27 bug

### CODE REVIEW

**Comment by catalinvr** [comment]
Seems that centos tests fails. When I will have time I will investigate this issue:
https://travis-ci.org/anuriq/chef-kibana5/jobs/346752842

**Comment by anuriq** [comment]
Hi @catalinvr 
Would you like to use [Upstart](http://upstart.ubuntu.com/)?

**Comment by catalinvr** [comment]
Hi,

For ubuntu 14.04 was easier for me to use the init.d template.
Do you have any recommendation regarding how I can implement kibana service?

Thanks,
Cata

File: CHANGELOG.md
Comment (anuriq) [inline CHANGELOG.md]: please, remove this change. Changelog is updated during version bump commit, which happens **after** merge

File: metadata.rb
Comment (anuriq) [inline metadata.rb]: please, remove this version bump. version update is made **after** PR merge

File: resources/configure.rb
Comment (anuriq) [inline resources/configure.rb]: if I am not wrong, this block should be 1 `else` later. Currently, for platforms other than ubuntu, service will not be configured.

File: templates/default/kibana.yml.erb
Comment (anuriq) [inline templates/default/kibana.yml.erb:94]: these are new options of Kibana configuration? what about backwards compatibility? what will change for current users?

**Review by anuriq — CHANGES_REQUESTED** [review]
please, review my comments

File: resources/configure.rb
Comment (catalinvr) [reply resources/configure.rb]: Thanks for this. Was my fault

File: templates/default/kibana.yml.erb
Comment (catalinvr) [reply templates/default/kibana.yml.erb:94]: Do you consider thats ok to keep both of them ?

Comment (catalinvr) [reply templates/default/kibana.yml.erb:94]: Hi @anuriq. Do you want to remove the new settings and to merge the pull request?

File: resources/configure.rb
Comment (mburns) [reply resources/configure.rb]: You can remove this whole block, since it is now duplicated below (and will catch for all platforms, including non-14.04 versions of Ubuntu).

File: templates/default/kibana.yml.erb
Comment (mburns) [reply templates/default/kibana.yml.erb]: Yes, please remove this commit and submit it as a separate PR so we can make sure to bump versions/defaults at the same time.

**Comment by anuriq** [comment]
Hi @catalinvr 
I am sorry to say this, but I cannot merge this feature. This cookbook was created to support only latest configurations, that's why only systemd is used for service start.
You are welcome to fork and use your own version. Thank you very much for your efforts.

**Pull request closed without merging**
